seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -5586,138 +5586,126 @@
-        "assert(convertToInteger(\"10011\") === 19, 'message: <code>convertToInteger(\"10011\")</code> should return 19');",
-        "assert(convertToInteger(\"111001\") === 57, 'message: <code>convertToInteger(\"111001\")</code> should return 57');",
-        "assert.isNaN(convertToInteger(\"JamesBond\"), 'message: <code>convertToInteger(\"JamesBond\")</code> should return NaN');"
+        "assert(checkEqual(1, 2) === false, 'message: <code>checkEqual(1, 2)</code> should return false');",
+        "assert(checkEqual(1, 1) === true, 'message: <code>checkEqual(1, 1)</code> should return true');",
+        "assert(checkEqual(1, -1) === false, 'message: <code>checkEqual(1, -1)</code> should return false');"
       ],
       "type": "waypoint",
       "challengeType": 1,
       "translations": {}
     },
     {
       "id": "587d7b7e367417b2b2512b21",
-      "title": "Use multiple Conditional Ternary Operators",
+      "title": "Use Multiple Conditional (Ternary) Operators",
       "description": [
-        "In the previous challenge, we used a single conditional operator. Now, we will learn how to use multiple conditionals.",
-        "For example:",
-        "<code>function findGreaterOrEqual(a, b, c) {</code>",
-        "<code>    if(a === b)</code>",
-        "<code>        return \"a and b are equal\";</code>",
-        "<code>    else if(a > b)</code>",
-        "<code>        return \"a is greater\";</code>",
-        "<code>    else</code>",
-        "<code>        return \"b is greater\";</code>",
-        "<code>}</code>",
-        "The above function can be written using multiple conditionals like:",
-        "<code>function findGreaterOrEqual(a, b) {</code>",
-        "<code>    return (a === b) ? \"a and b are equal\" : (a > b) ? \"a is greater\" : \"b is greater\";</code>",
-        "<code>}</code>",
-        "Note that the conditional operator is Right Assosiative.",
-        "Instructions",
-        "Using a Conditional Operator, check if a number is positive, negative or zero."
+        "In the previous challenge, you used a single <code>conditional operator</code>. You can also chain them together to check for multiple conditions.",
+        "The following function uses if, else if, and else statements to check multiple conditions:",
+        "<blockquote>function findGreaterOrEqual(a, b) {<br>  if(a === b) {<br>    return \"a and b are equal\";<br>  }<br>  else if(a > b) {<br>    return \"a is greater\";<br>  }<br>  else {<br>    return \"b is greater\";<br>  }<br>}</blockquote>",
+        "The above function can be re-written using multiple <code>conditional operators</code>:",
+        "<blockquote>function findGreaterOrEqual(a, b) {<br>  return (a === b) ? \"a and b are equal\" : (a > b) ? \"a is greater\" : \"b is greater\";<br>}</blockquote>",
+        "Note that the <code>conditional operator</code> is right associative.",
```

Comment:
I had to look this up when I read through these 😳  - if I can find a good link, I'll add it in, otherwise I can take this line out.